grpc_api.go

```diff
@@ -0,0 +1,159 @@
+/*
+ * File: grpc_api.go
+ *
+ * Description: various funcs needed for the gRPC API
+ */
+
+package main
+
+import (
+    "strconv"
+    pb "./lclusterpb"
+    "golang.org/x/net/context"
+)
+
+//! API function to start a job
+/*
+ * @param     Context            given host context
+ * @param     StartJobRequest    object to hold the start job request
+ *
+ * @return    StartJobResponse   the server's response to the remote call
+ * @return    error              error message, if any
+ */
+func (s *LclusterdServer) StartJob(ctx context.Context,
+  r *pb.StartJobRequest) (*pb.StartJobResponse, error) {
+
+    // Variable declaration
```

Comment:
yeah, i tend to be verbose, but those ones seems a bit generic; i went ahead and fixed 'em

https://github.com/rbisewski/lclusterd/pull/2/commits/6c4d657416d86af020eb7824d402869f4e8b067d